src/renderer/app.tsx

```diff
@@ -521,6 +528,16 @@ export class App extends React.Component<AppProps, AppState> {
   private updateLanguage = (): void => {
     this.props.langManager.updateContainer();
   }
+
+  private unmountBeforeClose = (): void => {
```

Comment:
I'm not sure what the purpose of this is, so please explain it to me.

This function makes refreshing the renderer (by focusing anything in the DevTools sidebar, and pressing F5) exit the launcher instead. I use this method to refresh the renderer heavily in development, to speed up iteration.